TFNetworkRecLayer.py

```diff
@@ -2600,7 +2600,9 @@ def search_resolve_cond(i, *args):
       acc_search_choices = SearchChoices(owner=latest_layer_choice, beam_size=latest_beam_size)
       final_choice_rec_vars = self.get_layer_rec_var_from_loop_vars(
         loop_vars=final_net_vars,
-        layer_name=latest_layer_choice.name)
+        layer_name=latest_layer_choice.name,
+        final_frame=True,
```

Comment:
This should have been set before, right? Or am I missing something?

Reply:
Hm yes, good question. Not sure anymore. But looks right. I'm not 100% sure about `seq_len`, if that comes from another beam.